src/file_dialog.rs

```diff
@@ -1097,13 +1116,16 @@ impl FileDialog {
 
             // Optionally, show a custom right panel (see `update_with_custom_right_panel`)
             if let Some(f) = right_panel_fn {
-                egui::SidePanel::right(self.window_id.with("right_panel"))
```

Comment:
is there a better solution?
this doesnt look that good I think

Reply:
I'm open to other approaches, but we have to make set width somehow...

Reply:
What is this about? That we save the right panel in a variable and then set variables dynamically using the builder methods? I believe there is no other way to do this.

Reply:
What is this about? That we save the right panel in a variable and then set variables dynamically using the builder methods? I believe there is no other way to do this.

Reply:
No, I don't know how we could change this. I'd leave it like this.